bounties/level-2/stellar-lab-darkmode.md

```diff
@@ -0,0 +1,40 @@
+# Create dark mode Theme for Stellar Lab
+
+| Hunter | Agent
+| :- | :-
+| 750-1500 XLM | 4 Credits
+
+## Description
+
+### What is this task?
+
+Our eyes are bleeding! Who is brave enough to help?  
+Implement a dark mode theme for the Stellar Lab similar to GitHub.
+
+750 XLM may be earned for the dark mode theme it self the other 750 for implementing a user choice between those two templates. 
```

Comment:
Implementing a toggle should take like 5 minutes so I say we just wrap it up together and have the whole bounty be to implement a dark mode with the ability to toggle between dark and light mode.